recipe/meta.yaml

```diff
@@ -19,17 +19,26 @@ requirements:
   build:
     - {{ compiler('c') }}
     - make
+    - cfdm
+    - cftime
+    - cfunits
+    - netcdf4
+    - netcdf-flattener
+    - numpy
+    - psutil
+    - packaging
+    - pyparsing
```

Comment:
Unless these are needed at build time we do not need to specify those here.

Reply:
When installing cf-python using pip these packages are installed - so they are all needed at build time (even if not directly needed by cf-python).

Reply:
When installing with pip you are at the  "run" phase and then yes, they are needed. But not at build time. Note that none of those are listed as setup requires in the package upstream.

Reply:
Thanks for the explanation. I've mostly worked with none-pip packages so far, so am not so familiar with the pip build system. I've reverted my changes here too - to see if only the makefile patch is needed to fix this issue.

Reply:
There are some pathological cases where the runtime dependencies are needed at build time when the package imports itself. That is a bad practice but quite common in old packages. Besides that, you only need the setup requires dependencies at build time (and the non-python ones that are hard to specify using python metadata.)